### ISSUE TITLE
[Package Request]: Rem0o.FanControl

### ISSUE THREAD

**Issue by D3strukt0r** [microsoft/winget-pkgs#137769]
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/Rem0o/FanControl.Releases/releases/tag/V180
Publisher: Rem0o
Package Name: Rem0o.FanControl
Description: A focused and highly customizable fan controlling software for Windows.
Package Version: 180
Installer URL:
- https://github.com/Rem0o/FanControl.Releases/releases/download/V180/FanControl_net_8_0.zip (Preferred)
- https://github.com/Rem0o/FanControl.Releases/releases/download/V180/FanControl_net_7_0.zip
- https://github.com/Rem0o/FanControl.Releases/releases/download/V180/FanControl_net_4_8.zip

**Comment by SpecterShell** [comment]
This package has already been added. Try `winget install Rem0o.FanControl`.

**Comment by Trenly** [comment]
[Policy] Unblocked; Will require `ManifestVersion: 1.9.0` with `ArchiveBinariesDependOnPath: true`

**Comment by D3strukt0r** [comment]
@DandelionSprout yeah seems like this was solved a long time ago, just noticed he started adding installers. will close
